solutions/standard/main.tf

```diff
@@ -64,11 +95,12 @@ module "kms" {
 
 module "elasticsearch" {
   source                        = "../../modules/fscloud"
+  depends_on                    = [time_sleep.wait_for_authorization_policy]
   resource_group_id             = module.resource_group.resource_group_id
   name                          = var.prefix != null ? "${var.prefix}-${var.name}" : var.name
   region                        = var.region
   plan                          = var.plan
-  skip_iam_authorization_policy = var.skip_iam_authorization_policy
+  skip_iam_authorization_policy = var.skip_iam_authorization_policy || local.create_cross_account_auth_policy
```

Comment:
This is different to redis. I am wonder if redis needs updating rather than elastic search? Could you either remove this or raise an issue/PR to fix redis.  MTIA.

Reply:
I will add this to redis PR that I am working on